### PR TITLE
channel and logo additions

### PR DESCRIPTION
**Pull request by Niyasmkkv**

### CODE REVIEW

File: channels/in.m3u [hunk 1]
```diff
@@ -379,9 +381,13 @@ http://103.199.160.85/Content/puthuyugam/Live/Channel(Puthuyugam)/index.m3u8
 #EXTINF:-1 tvg-id="RPlusNews.in" tvg-name="R Plus News" tvg-country="IN" tvg-language="" tvg-logo="" group-title="News",R Plus News (360p)
 https://cdn.smartstream.video/smartstream-us/rplus/rplus/playlist.m3u8
 #EXTINF:-1 tvg-id="RajDigitalPlus.in" tvg-name="Raj Digital Plus" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="Movies",Raj Digital Plus (404p)
-http://acv.asianetmobiletvplus.com/channels/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_hls/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_master.m3u8
+http://acv.asianetmobiletvplus.com/channels/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_hls/rhttp://198.144.149.82:8080/NOTV/RELAXTV/index.m3u8?token=GTRajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_master.m3u8
```
Comment (freearhey) [inline channels/in.m3u:384]: The link is not valid.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

Everything else looks good to me.

File: channels/in.m3u (hunk 1 above)
Comment (Niyasmkkv) [reply channels/in.m3u:384]: Delete the link then.

**Comment by Sphinxroot** [comment]
not the same channel  Raj Digital Plus (404p)  and your adding Relax TV

#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="http://expressott.in/uploads/tv_image/relax-tv.png" group-title="",Relax TV
http://198.144.149.82:8080/NOTV/RELAXTV/index.m3u8?token=GTR

File: channels/in.m3u [hunk 2]
```diff
@@ -379,9 +381,13 @@ http://103.199.160.85/Content/puthuyugam/Live/Channel(Puthuyugam)/index.m3u8
 #EXTINF:-1 tvg-id="RPlusNews.in" tvg-name="R Plus News" tvg-country="IN" tvg-language="" tvg-logo="" group-title="News",R Plus News (360p)
 https://cdn.smartstream.video/smartstream-us/rplus/rplus/playlist.m3u8
 #EXTINF:-1 tvg-id="RajDigitalPlus.in" tvg-name="Raj Digital Plus" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="Movies",Raj Digital Plus (404p)
-http://acv.asianetmobiletvplus.com/channels/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_hls/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_master.m3u8
+http://198.144.149.82:8080/NOTV/RELAXTV/index.m3u8?token=GTRajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_master.m3u8
```
Comment (freearhey) [inline channels/in.m3u:384]: Still not working.
image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/in.m3u [hunk 3]
```diff
@@ -381,7 +381,7 @@ http://103.199.160.85/Content/puthuyugam/Live/Channel(Puthuyugam)/index.m3u8
 #EXTINF:-1 tvg-id="RPlusNews.in" tvg-name="R Plus News" tvg-country="IN" tvg-language="" tvg-logo="" group-title="News",R Plus News (360p)
 https://cdn.smartstream.video/smartstream-us/rplus/rplus/playlist.m3u8
 #EXTINF:-1 tvg-id="RajDigitalPlus.in" tvg-name="Raj Digital Plus" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="Movies",Raj Digital Plus (404p)
-http://198.144.149.82:8080/NOTV/RELAXTV/index.m3u8?token=GTRajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_master.m3u8
+http://acv.asianetmobiletvplus.com/channels/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_hls
```
Comment (freearhey) [inline channels/in.m3u:384]: Same thing.
image

**Review by freearhey — DISMISSED** [review]
To be sure the link works you can first test it via VLC player or a site like [streamtest.in](https://streamtest.in/)

**Comment by Niyasmkkv** [comment]
I think I accidentally copied the other channel's link with that, this link was already broken. Never had Raj Digital Plus' link with me.

**Comment by freearhey** [comment]
In this case, just put the old link back and everything will be fine.

**Comment by Niyasmkkv** [comment]
I did put the old link back. That channel was broken.

File: channels/in.m3u (hunk 3 above)
Comment (freearhey) [reply channels/in.m3u:384]: this is an old link
```suggestion
http://acv.asianetmobiletvplus.com/channels/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_hls/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_master.m3u8
```

File: channels/in.m3u [hunk 4]
```diff
@@ -379,9 +381,13 @@ http://103.199.160.85/Content/puthuyugam/Live/Channel(Puthuyugam)/index.m3u8
...
 #EXTINF:-1 tvg-id="RajDigitalPlus.in" tvg-name="Raj Digital Plus" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="Movies",Raj Digital Plus (404p)
-http://acv.asianetmobiletvplus.com/channels/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_hls/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_master.m3u8
+http://acv.asianetmobiletvplus.com/channels/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_hls
+#EXTINF:-1 tvg-id="RealTV.in" tvg-name="Real TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="http://expressott.in/uploads/tv_image/real-tv.jpg" group-title="Music",Real TV (1080p)
+http://cloud.logicwebs.in:1935/realtv/realtv1/playlist.m3u8
 #EXTINF:-1 tvg-id="RealNewsKerala.in" tvg-name="Real News Kerala" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/EmBpOWM.png" group-title="News",Real News Kerala (1080p)
 https://bk7l298nyx53-hls-live.5centscdn.com/realnews/e7dee419f91aa9e65939d3677fb9c4f5.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="RelaxTV" tvg-name="Rlax TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="http://expressott.in/uploads/tv_image/relax-tv.png" group-title="Music",Relax TV
```
Comment (Dum4G) [inline channels/in.m3u:389]:
```suggestion
#EXTINF:-1 tvg-id="RelaxTV" tvg-name="Relax TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="http://expressott.in/uploads/tv_image/relax-tv.png" group-title="Music",Relax TV
```

**Review by Dum4G — APPROVED** [review]

**Pull request merged**